packages/server-wallet/src/__test__/create-and-close-channel/direct-funding.test.ts

```diff
@@ -93,6 +94,11 @@ it('Create a directly funded channel between two wallets ', async () => {
     turnNum: 0,
   });
 
+  // Let's check what wallet a is waiting for. Expect it to be 'theirPreFundSetup' or ''
+  // (if the objective has never been cranked)
+  const waitingForBefore = (await a.getObjective(`OpenChannel-${channelId}`)).waitingFor;
+  expect([WaitingFor.theirPreFundSetup, '']).toContain(waitingForBefore);
```

Comment:
Is `''''` a valid value anymore?

Reply:
Good catch, this should be changed to `null`.